apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -1176,6 +1177,10 @@ type DatadogAgentComponentOverride struct {
 	// +optional
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
 
+	// The deployment strategy to use to replace existing pods with new ones.
+	// +optional
+	Strategy *UpdateStrategy `json:"strategy,omitempty"`
```

Comment:
```suggestion
	UpdateStrategy *UpdateStrategy `json:"updateStrategy,omitempty"`
```
I think this would be clearer